packages/graph/hash_graph/lib/graph/src/api/rest/entity.rs

```diff
@@ -161,6 +162,14 @@ async fn get_entities_by_query<P: StorePool + Send>(
             StatusCode::INTERNAL_SERVER_ERROR
         })
         .and_then(|store| async move {
+            let mut query = NewStructuralQuery::try_from(query).map_err(|error| {
+                tracing::error!(?error, "Could not deserialize query");
```

Comment:
Given the `try_from`s here are _very_ temporary, I am not gonna nitpick on the error strings :+1: